packages/@aws-cdk/aws-ec2-alpha/README.md

```diff
@@ -231,6 +231,42 @@ new Route(this, 'DynamoDBRoute', {
 });
 ```
 
+## VPC Peering Connection
+
+VPC peering connection allows you to connect two VPCs and route traffic between them using private IP addresses. The VpcV2 construct supports creating VPC peering connections through the `VPCPeeringConnection` construct from the `route` module.
+
+Here's an example of how to create a VPC peering connection between two VPCs:
+
+```ts
+const stack = new Stack();
+
+const vpcA = new VpcV2(this, 'VpcA', {
+  primaryAddressBlock: IpAddresses.ipv4('10.0.0.0/16'),
+});
+
+const vpcB = new VpcV2(this, 'VpcB', {
+  primaryAddressBlock: IpAddresses.ipv4('10.1.0.0/16'),
+});
+
+const peeringConnection = new VPCPeeringConnection(this, 'crossAccountCrossRegionPeering', {
+  isCrossAccount: true,
+  requestorVpc: vpcA,
+  acceptorVpc: vpcB,
+  acceptorAccountId: '123456789012',
+  acceptorRegion: 'us-west-2',
+});
```

Comment:
I think this is not accurate example, these VPCs are created in the CDK application which means both are created in the same account, and region, so `isCrossAccount`, `acceptorAccountId`, and `acceptorRegion` properties are not required.

Reply:
Updated design to what we discussed in the [meeting](https://quip-amazon.com/eBSvAUhlrV4e/CDK-VPC-Peering-Connection-Design-Doc). Commit that makes the change: [2b4c5c6](https://github.com/aws/aws-cdk/pull/31645/commits/2b4c5c6a80d0bb26306b6d94826113a1580e6846)